.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "abstractions-xunit"]
 	path = src/abstractions-xunit
 	url = https://github.com/xunit/abstractions.xunit
+[submodule "src/docker-creds-provider-8.0.2xx"]
+	path = src/docker-creds-provider-8.0.2xx
```

Comment:
I was expecting the submodule to be versioned in terms of it's package rather than what version of the sdk depends on it.  The use of the SDK version can be confusing if more than one band/version depends on it.

I also think the "other" submodule should be versioned as well - e.g. 2.2.0

This is what was done in the past when multiple newtonsoft versions were supported.

Reply:
I considered that but I didn't really like it because it would require updating the submodule name for each new version update. I see your point though about multiple bands depending on it.

Reply:
As soon as you have multiple versions, I struggle to see any option that doesn't require all versions to use a version number in their name.  Seeing 8.0.2xx in main doesn't make sense.  8.0.2xx is not an LTS band so it will reach EOL and need to be renamed.

The patterns used back when we had multiple Newtonsoft versions were proven.  Yes they have some cons but IMO they are better than using SDK versions and a non-versioned folder for one.